includes/ajax-functions.php

```diff
@@ -538,9 +538,15 @@ function edd_ajax_get_states_field() {
 	// Maybe setup the new listbox.
 	if ( ! empty( $states ) ) {
 		$field_name = sanitize_text_field( $_POST['field_name'] );
+		$field_id   = sanitize_text_field( $_POST['field_id'] );
```

Comment:
This causes the following notice when adding a tax rate in settings

```
PHP Notice:  Undefined index: field_id in /var/www/edd3/wp-content/plugins/easy-digital-downloads/includes/ajax-functions.php on line 541
```

1. Go to Downloads > Settings > Taxes.
2. Select a country via the dropdown.
3. Notice triggers when states are loaded for that country.

Reply:
@ashleyfae Updated. I also added a check and fallback for `field_name` which should never be empty but could be with incorrect implementation.